### ISSUE TITLE
flexbox mat-password-strength-info div.info-row wrong alignment

### ISSUE THREAD

**Issue by andreElrico** [angular-material-extensions/password-strength#158]
have you tried your awesome <3 password module on OPERA?

The items inside `mat-password-strength-info` where misaligned for me.

adding the following css in global styles.scss solved it.
```
mat-password-strength-info > mat-card  > mat-card-content > div.info-row {
    align-items: center;
}
```
I guess it has something to do with different default values of different browsers.

Lets see :-)

**Comment by AnthonyNahas** [comment]
I've just tested it with opera and it's working fine
<img width="298" alt="Bildschirmfoto 2019-04-11 um 20 56 27" src="https://user-images.githubusercontent.com/20149774/55983836-4e4f8a80-5c9c-11e9-9547-e13f4220df22.png">

**Comment by AnthonyNahas** [comment]
this has been fixed in `v3.8.0` that will be published in few minutes ;)

**Comment by AnthonyNahas** [comment]
[v3.8.0](https://www.npmjs.com/package/@angular-material-extensions/password-strength) is now live 👍
